src/devices/xiaomi.js

```diff
@@ -1026,7 +1026,7 @@ module.exports = [
         model: 'WS-USC03',
         vendor: 'Xiaomi',
         description: 'Aqara smart wall switch (neutral, single rocker)',
-        fromZigbee: [fz.on_off, fz.xiaomi_power, fz.aqara_opple],
+        fromZigbee: [fz.on_off, fz.xiaomi_power, fz.xiaomi_multistate_action, fz.aqara_opple],
```

Comment:
Please also update the actions in `e.action(` below

Reply:
Hi Koenkk, the action was already exposed in my previous commit. This device supports two actions: single, double. I just forgot to add the converter to translate the actions.

Reply:
I see, thanks!